orm/db.go

```diff
@@ -488,6 +488,115 @@ func (d *dbBase) InsertValue(q dbQuerier, mi *modelInfo, isMulti bool, names []s
 	return id, err
 }
 
+// InsertOrUpdate a row
+// If your primary key or unique column conflict will update
+// If no will insert
+func (d *dbBase) InsertOrUpdate(q dbQuerier, mi *modelInfo, ind reflect.Value, a *alias, args ...string) (int64, error) {
+	iouStr := ""
+	mysql := DRMySQL
+	postgres := DRPostgres
+	driver := a.Driver
+	argsMap := map[string]string{}
+	args0 := ""
+	if driver == mysql {
+		iouStr = "ON DUPLICATE KEY UPDATE"
+	} else if driver == postgres {
+		if len(args) == 0 || (len(strings.Split(args0, "=")) != 1) {
+			return 0, fmt.Errorf("`%s` use insert or update must have a conflict column arg in first", a.DriverName)
+		} else {
+			args0 = strings.ToLower(args[0])
+			iouStr = fmt.Sprintf("ON CONFLICT (%s) DO UPDATE SET", args0)
```

Comment:
I  think you can just use `iouStr = fmt.Sprintf("ON CONFLICT (%s) DO UPDATE SET", strings.ToLower(args[0]))`